bokehjs/src/lib/models/glyphs/webgl/regl_wrap.ts

```diff
@@ -368,41 +323,45 @@ function regl_marker(regl: any, marker_type: MarkerType): ReglRenderFunction {
         buffer: regl.buffer([[-0.5, -0.5], [-0.5, 0.5], [0.5, 0.5], [0.5, -0.5]]),
         divisor: 0,
       },
-      a_center: {
-        buffer: regl.prop('center'),
-        divisor: 1,
+      a_center: (_: Context, props: Props) => {
+        return {
+          buffer: regl.buffer(props.center),
+          divisor: 1,
+        }
       },
-      a_size: (_: any, props: any) => {
-        return one_each_or_constant(props.size, 1, false, props.nmarkers)
+      a_size: (_: Context, props: Props) => {
```

Comment:
All of those type annotations are unnecessary given you've already provided those types as generics to `DrawConfig`. Also you may want to consider using `a_size(_, props) { ... }` function shorthand syntax in objects.

Reply:
Having started adding types it is difficult to stop doing it everywhere, even if not really needed!

I've implemented both of your suggestions and the code is definitely simpler and easier to follow.  I did wonder if I needed to keep the `Context` types as they are not part of the `DrawConfig` generics, but there are no compiler complaints without them.